eox_tenant/settings/aws.py

```diff
@@ -43,3 +47,5 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
             'eox_tenant.middleware.AvailableScreenMiddleware',
             'eox_tenant.middleware.MicrositeCrossBrandingFilterMiddleware',
         ]
+
+        settings.AUTHENTICATION_BACKENDS = ['eox_tenant.auth.TenantAwareAuthBackend'] + list(settings.AUTHENTICATION_BACKENDS)  # pylint: disable=line-too-long
```

Comment:
@jfavellar90 I am not sure how I should do this. 
I only need to replace the 'openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends.EdxRateLimitedAllowAllUsersModelBackend' with 'eox_tenant.auth.TenantAwareAuthBackend'